changelog/323.feature.rst

```diff
@@ -0,0 +1 @@
+Introduce concept of global coordinates
```

Comment:
This should be slightly more descriptive.  For example, `Introduce a new class, GlobalCoords, for holding scalar coordinates that don't apply to any pixel axes.`

Reply:
Sure, let me fix it soon.

Reply:
Updated